modules/Examples/error-framework/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+ev_add_module(ExampleErrorRaiser)
+ev_add_module(ExampleErrorSubscriber)
+ev_add_module(ExampleErrorGlobalSubscriber)
+
+if(EVEREST_ENABLE_PY_SUPPORT)
```

Comment:
This should not be necessary since ev_add_module already checks if the language support of the module is activated or not. Did you have issues with this? Then the check in everest-generate.cmake would be broken and needs fixing